### DISCUSSION TITLE
how to host this siteon github

### DISCUSSION THREAD

**Discussion by varghesejose2020** [onweru/compose#47] [Q&A]
how to host  this  hugo site on github

**Comment by onweru** [comment]
Use GitHub actions https://discourse.gohugo.io/t/publish-your-next-hugo-blog-post-automatically-while-you-sleep-using-github-actions/27438

**Reply by varghesejose2020** [reply]
any other way like github pages

**Reply by varghesejose2020** [reply]
i am not familiar with github action

**Reply by onweru** [reply]
Github actions allow you to host using Github pages. I included a link guiding you through that process. Kindly read through.

**Reply by hasthamalp** [reply]
![image](https://user-images.githubusercontent.com/72604216/125153845-29e10200-e174-11eb-8c3f-6ebd54e187ae.png)

how to resolve these

**Reply by mysubcult** [reply]
> ![image](https://user-images.githubusercontent.com/72604216/125153845-29e10200-e174-11eb-8c3f-6ebd54e187ae.png)
> 
> how to resolve these

same problem, how to solve it?

**Reply by mysubcult** [reply]
@onweru help please.

**Reply by onweru** [reply]
@mysubcult, can I have a look at your repo?
